integr8ly/integreatly-operator
# use non-colliding package names

## verification
- Have a cluster with a [certified](https://gist.githubusercontent.com/philbrookes/4552d79fa7ca18f2516825b0a65b3bd3/raw/8e21c43dfef799e192b4e6eb589627779a8d36e8/certified.yaml), a [community](https://gist.githubusercontent.com/philbrookes/4552d79fa7ca18f2516825b0a65b3bd3/raw/8e21c43dfef799e192b4e6eb589627779a8d36e8/community.yaml) and a [Red Hat](https://gist.githubusercontent.com/philbrookes/4552d79fa7ca18f2516825b0a65b3bd3/raw/8e21c43dfef799e192b4e6eb589627779a8d36e8/redhat.yaml) operatorsource
- add an operatorsource pointing at philbrookes
- provision integreatly-operator into a test namespace
- create an installation CR
- see everything come up except 3scale. (3scale fails for unrelated reasons)
